Automattic/jetpack
# [Plugin] Backup: Updated styling wrapping ConnectScreen

#### Changes proposed in this Pull Request:

* Updated the styles wrapping the ConnectScreen to more closely match design

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
* Install the Backup plugin on an unconnected site
* If you already have a site with a connection you can break it with the Jetpack Debug tools
* Ensure the ConnectScreen has no Header/Footer and is not full on desktop

Before patch:
<img width="2552" alt="Screen Shot 2021-07-20 at 12 27 30 PM" src="https://user-images.githubusercontent.com/1992658/126360964-f60a350e-19c6-4b13-a474-85d5acfb4e9c.png">

After patch:
<img width="1458" alt="Screen Shot 2021-07-20 at 12 17 22 PM" src="https://user-images.githubusercontent.com/1992658/126360211-e838c870-e32e-421f-bf9a-3f0ff5914a5a.png">
